feat: improved tracability to study problems with issuing

marnixdessing opened the pull request:
- Add request id to statistics log entry to correlate it to the lambda invocation
- Add yivi session token to correlate to yivi container events

marnixdessing committed:
feat: improved tracability for statistics and session in container

marnixdessing committed:
fix: remove hashing as we need to correlate from webapp to server

marnixdessing committed:
chore: better comments

marnixdessing committed:
Merge pull request #654 from GemeenteNijmegen/feat/tracability-improvements

feat: improved tracability for statistics and session in container

marnixdessing committed:
Merge branch 'production' into acceptance

WietekeNijmegen reviewed (dismissed):
Al gezamenlijke review gedaan toen het naar acc ging. Getest op acceptatie, lijkt goed te gaan. Heb geen foutscenario kunnen testen aldaar

Commit:
chore(deps): upgrade dependencies

Upgrades project dependencies. See details in [workflow run].

[Workflow Run]: https://github.com/GemeenteNijmegen/yivi-issue-app/actions/runs/9216643007

------

*Automatically created by projen via the "upgrade-acceptance" workflow*

Signed-off-by: github-actions <github-actions@github.com>

github-actions[bot] committed:
Merge pull request #657 from GemeenteNijmegen/github-actions/upgrade-acceptance

chore(deps): upgrade dependencies

marnixdessing committed:
chore: trigger builds
